tests/unit/python/execution_tree/eval.py

```diff
@@ -40,10 +40,19 @@ def fib(n):
     else:
         return fib(n-1)+fib(n-2)
 
+assert fib.__physl_src__ == \
+    'block#1#0(define#1#0(fib#1#0, n#1#8, ' + \
+        'if(n#2#7 < 2, n#3#15, ' + \
+            '(fib((n#5#19 - 1)) + fib((n#5#28 - 2))))' + \
+        '), fib#1#0)\n'
+
 assert fib(10)[0] == 55.0
```

Comment:
Probably you want to run this change before you replace the source as well as after. Otherwise, you're disabling the previous test.

Reply:
Sorry, I don't know what you're referring to - care to elaborate?

Reply:
@stevenrbrandt could you elaborate, please?